src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanel.js

```diff
@@ -65,12 +65,17 @@ class DocumentPanel extends Component {
                         id="DocumentPanel.AfterAuthors"
                         metadata={doc.metadata}
                       />
+                      <div
+                        className={`default-margin-bottom`}>
+                        Published by <b>{doc.metadata.imprint?.publisher}</b>
```

Comment:
We should check if the publisher exists before displaying it (as you did above) since it's not a mandatory field

Reply:
Since there is the previous `{doc.metadata.imprint?.publisher ?`, inside the if will be for sure there, so you can:
`Published by <b>{doc.metadata.imprint?.publisher}</b>` -> `Published by <b>{doc.metadata.imprint.publisher}</b>`